src/runtime_src/core/tools/xbreplay/src/utils/cmd_args.cpp

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#include "utils/cmd_args.hpp"
+
+namespace xrt::tools::xbreplay {
+/*
+ * This funciton is used to print replay usage.
+ */
+void cmd_args::print_usage()
+{
+  std::cout << "Please find below usage information" << std::endl;
```

Comment:
```suggestion
  std::cout << "Please find below usage information" << `\n`
```
Please pretend `std::endl` doesn't exists